clients/unshaded/src/main/java/tachyon/client/file/options/CreateOptions.java

```diff
@@ -143,6 +143,20 @@ public UnderStorageType getUnderStorageType() {
   }
 
   /**
+   * @return the name : value pairs for all the fields
+   */
+  @Override
+  public String toString() {
+    StringBuilder sb = new StringBuilder("CreateOptions(");
+    sb.append(super.toString()).append(", mBlockSizeBytes: ").append(mBlockSizeBytes);
```

Comment:
Remove the leading `m` from all the descriptions. This is used by the implementation to distinguish member variables, but there is no need to propagate them to logs.